### DISCUSSION TITLE
Can I have a different logo on light/dark backgrounds in reveals?

### DISCUSSION THREAD

**Discussion by EllaKaye** [quarto-dev/quarto-cli#9820] [Q&A]
### Description

Hello. I have two versions of a logo, identical except that one is dark and one is white. I also have a custom .scss which includes an `{.inverse}` class, which, amongst other things, sets the background to a dark colour.

In my yaml, I have 

```
format: 
  revealjs: 
    logo: logo-dark.png
```

This dark logo looks great against the regular white background. But, on my `{.inverse}` slides, it gets mostly lost. Is there a way I can swap the logo to `logo-white.png` whenever the inverse class is applied?

I tried something similar to this trick, #1647, using the selector `body > div.reveal.has-logo.none.focused.has-vertical-slides.has-horizontal-slides.ready > img`, but it didn't work. 

If I can't swap the logo, I'd like to be able to turn it off when {.inverse} is applied, or when the background is in any other way identified to be 'dark'? I think that may be related to this (open) issue: #1445

**Comment by mcanouil** [comment]
See:
- #3417

**Comment by EllaKaye** [comment]
Thanks @mcanouil. I forgot to say in my original post that I had seen that discussion but for some reason @sagkazarmark's workaround isn't working for me. In the meantime, I've decided not to use the logo after all, so happy to drop this for now. If I change my mind, I'll take a closer look at your JS solution.

**Reply by mcanouil** [reply]
The link was mostly to avoid having the discussion on several threads.
